### PR TITLE
New package releases

### PR DESCRIPTION
**Pull request by jessebraham**
I think we should be good to go, but last chance to change/fix anything before the release.

### CODE REVIEW

**Comment by jessebraham** [comment]
I don't want to deal with the broken `CHANGELOG.md` checks so I am just ignore the faiure 🤷🏻‍♂️

File: esp-hal/CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.19.0] - 2024-06-15
```
Comment (JurajSadel) [inline esp-hal/CHANGELOG.md:8]:
```suggestion
## [0.19.0] - 2024-07-15
```
Typo?

File: examples/src/bin/embassy_usb_serial.rs [hunk 2]
```diff
@@ -80,7 +80,6 @@ async fn main(_spawner: Spawner) -> () {
 
     // Create embassy-usb DeviceBuilder using the driver and config.
     // It needs some buffers for building the descriptors.
-    let mut device_descriptor = [0; 256];
```
Comment (JurajSadel) [inline examples/src/bin/embassy_usb_serial.rs:83]: Just to double check, is this change intended?

File: examples/src/bin/embassy_usb_serial.rs [hunk 3]
```diff
@@ -90,7 +89,6 @@ async fn main(_spawner: Spawner) -> () {
     let mut builder = Builder::new(
         driver,
         config,
-        &mut device_descriptor,
```
Comment (JurajSadel) [inline examples/src/bin/embassy_usb_serial.rs:93]: Same as above.

**Review by JurajSadel — CHANGES_REQUESTED** [review]
Mostly LGTM, I left a suggestion on a typo and two comments just to be sure, if the changes are intended or not. Thanks!

**Comment by jessebraham** [comment]
> This is a nitpick, but can we get the fix introduced in [esp-rs/xtensa-lx@67d5856](https://github.com/esp-rs/xtensa-lx/commit/67d5856bc004c965ad799ffcbe5473319be26755) in this release.

I've published a new patch release (`0.2.2`) of the `xtensa-lx-rt-procmacros` package, so that should hopefully be adequate.

**Review by JurajSadel — APPROVED** [review]
LGTM, thanks!

**Review by MabezDev — APPROVED** [review]
LGTM, thanks!
